### ISSUE TITLE
Is it possible to specify a font?

### ISSUE THREAD

**Issue by devilgate** [mmanela/MarkdownOutlook#13]
Outlook 2010 on Win 7. Both the preview and the sent messagess are in horrible Times New Roman, rather than any font I might be using. Is there any way to specify the font?

Also, it only works at all if you set the message format to "Rich Text". is that as desgned?

**Comment by mmanela** [comment]
I imagine it is possible, we would probably need to embed the style as part of the generated HTML. If you set the default font for the mail message does it not apply it?

**Comment by devilgate** [comment]
No. I just checked that I had the default font set to Verdana, as [documented by Microsoft](https://support.office.microsoft.com/en-us/article/Change-the-default-font-or-text-color-for-email-messages-1aabb236-01d4-4faf-b998-a4087da3ceab?CorrelationId=dc87262f-532e-42c9-9715-f70fa049956e&ui=en-US&rs=en-US&ad=US), and I still get TNR when I use the MarkdownOutlook plugin.

I can show you screen grabs if it will help.

**Comment by airstrike** [comment]
A broader alternative to this issue is referencing a .css file that styles the rendered e-mail, with a default one being provided in the instalation. A killer feature, IMHO!

**Comment by TheMightyLlama** [comment]
Yep, same issue here. Installed it for work and found that Times New Roman was default. I'd love to have a css location so I can style my own emails.

**Comment by mmanela** [comment]
This would be great to add. I currently do not have the time to play with this but I would gladly accept a PR.
